StepDefinitions/MarsLanguageSkillsTestsStepDefinitions.cs

```diff
@@ -0,0 +1,115 @@
+using Mars_Language_Skills.Pages;
+using MarsLanguageSkillsSpecflow.Pages;
+using MarsLanguageSkillsSpecflow.Utilities;
+using OpenQA.Selenium.Chrome;
+
+namespace MarsLanguageSkillsSpecflow.StepDefinitions
+{
+    [Binding]
+    public class MarsLanguageSkillsTestsStepDefinitions : CommonDriver
+    {
+        LoginPage loginPageObj = new LoginPage();
+        ProfilePage profilePageObj = new ProfilePage();
+        LanguageFeature languageFeatureObj = new LanguageFeature();
+        SkillFeature SkillFeatureObj = new SkillFeature();
+
+
+        [Given(@"User logs in Mars portal")]
+        public void GivenUserLogsInMarsPortal()
+        {
+            // IWebDriver driver = new ChromeDriver();
+            driver = new ChromeDriver();
+
+            //LoginPage obeject initialization
+
+           // LoginPage loginPageObj = new LoginPage();
+            loginPageObj.LoginActions(driver, "krits17.kt@gmail.com", "Singapore_24");
+           
+        }
+
+        [When(@"User navigates to profile page")]
+        public void WhenUserNavigatesToProfilePage()
+        {
+            profilePageObj.VerifyLoggedinUser(driver);
+        }
+
+        [When(@"User adds a language with level, and clicks Add button")]
+        public void WhenUserAddsALanguageWithLevelAndClicksAddButton()
```

Comment:
you need to think about state 

example before adding language you need to make sure there a no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing language so that the state is reset, this will guarantee that your test does not produce false negatives and passes always
you can assume that test execution order is not always guaranteed
imagine if you run the tests  in parallel mode the all tests will run at the same time
so you need to also make sure you do not wipe out other tests data
hence why when you remove data the best thing to do is delete only data you added for the test to be correct
e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2

So to sum up you should delete all data if there is any before the start of any test and then after each test you should delete the data that a particular test is adding!
This is ensure that upon staring the test pack you have reset state and upon finishing each test deletes their data that they added!

also say you are updating a language then you must added language then update it do not assume the language is there!



so think about first clean up before any test runs (you delete all data around this feature e.g  before any language test run you delete all languages). Then after each test delete only data that the test has added! (test1 added english and test2 to added french then test1 should only delete english and test2 should delete french only)